data/model/team.py

```diff
@@ -65,12 +65,10 @@ def create_team(name, org_obj, team_role_name, description=""):
 
 
 def add_user_to_team(user_obj, team):
-    try:
-        return TeamMember.create(user=user_obj, team=team)
-    except Exception:
-        raise UserAlreadyInTeam(
```

Comment:
Is there code that expects to catch this `UserAlreadyInTeam` error?

Reply:
yes, [this](https://github.com/quay/quay/blob/master/endpoints/api/team.py#L441) method is expected to throw 400 when user is already a member of the team. I moved up this exception in [this](https://github.com/quay/quay/pull/3550/commits/7e458e8eeb45682a25c88ea82c87d397b35ac65c) commit.